lib/vp8.py

```diff
@@ -90,3 +92,17 @@ def ResultData(self, encodedfile):
     more_results = {}
     more_results['frame'] = file_codec.MatroskaFrameInfo(encodedfile)
     return more_results
+
+  def EncoderVersion(self):
+    # The vpxenc command line tool outputs the version number of the
+    # encoder as part of its error message on illegal arguments.
+    try:
+      subprocess.check_output([encoder.Tool('vpxenc')],
```

Comment:
If this function doesn't raise, this method will return None which is quite weird. Maybe raise or assert outside the try-except block, like 

assert false, 'Expected vpxenc to return error code when called without arguments'